dbt/include/bigquery/macros/adapters.sql

```diff
@@ -194,3 +193,24 @@
   {%- endcall %}
 
 {% endmacro %}
+
+
+{% macro bigquery__test_unique(model, column_name) %}
```

Comment:
Is this the correct file for this macro to be to?

Reply:
Sure, this works!

It could also go into its own file, something like `dbt/include/bigquery/macros/generic_tests/unique.sql`. There's no functional difference, it's just a question of organization.